front/src/modules/trainschedule/components/ManageTrainSchedule/PacedTrainSettings.tsx

```diff
@@ -9,13 +11,22 @@ import {
   getCadence,
 } from 'reducers/osrdconf/operationalStudiesConf/selectors';
 import { useAppDispatch } from 'store';
+import { Duration } from 'utils/duration';
 
 const PacedTrainSettings = () => {
-  const timeRangeDuration = useSelector(getTimeRangeDuration);
-  const cadence = useSelector(getCadence);
+  const timeRangeDurationStore = useSelector(getTimeRangeDuration);
+  const cadenceStore = useSelector(getCadence);
```

Comment:
Hum do you think we can do this ?
```suggestion
  const timeRangeDurationStore = useSelector(getTimeRangeDuration).total('minute');
  const cadenceStore = useSelector(getCadence).total('minute');
```

Reply:
Looks like it's working yes !